spec/controllers/admin/versions_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'spec_helper'
+
+describe Admin::VersionsController do
+
+  let!(:conference) { create(:conference, short_title: 'exampletitle', description: 'Example Description') }
+  let(:admin) { create(:admin) }
+
+  with_versioning do
+    describe 'GET #revert' do
+      before :each do
+        sign_in admin
+      end
+
+      context 'when no attribute is set' do
+        it 'reverts all changes for update events' do
```

Comment:
update events is confusing, it makes me think of updating events, which is not what you are testing for here.
`update action` perhaps?